addons/website_sale_stock/models/product_template.py

```diff
@@ -3,6 +3,7 @@
 from odoo import api, fields, models
 from odoo.http import request
 from odoo.tools.translate import html_translate
+from odoo.tools import float_round
```

Comment:
```suggestion
from odoo.tools import float_round
from odoo.tools.translate import html_translate
```
I'd recommend looking into installing `ruff` as a linter using [this config](https://github.com/odoo/odoo/blob/master/ruff.toml) to indicate whether imports are ordered & grouped correctly.